plugins/modules/ssm_parameter.py

```diff
@@ -137,6 +142,29 @@
 - name: recommend to use with aws_ssm lookup plugin
   ansible.builtin.debug:
     msg: "{{ lookup('amazon.aws.aws_ssm', 'Hello') }}"
+
+- name: Create or update key/value pair in AWS SSM parameter store w/ tags
+  community.aws.ssm_paramater:
```

Comment:
```suggestion
  community.aws.ssm_parameter:
```

Reply:
I think the PR must just be rebased now: https://github.com/ansible-collections/community.aws/pull/1644